keyform/templates/keyform/emails/status_update.html

```diff
@@ -0,0 +1,422 @@
+                                                                    </tr>
+                                                                    <tr>
+                                                                        <td>{% trans 'Room' %} #'s:</td>
+                                                                        {% for key in request.keydata_set.all %}
+                                                                            <td>{{ key.room_number }}</td>
+                                                                        {% endfor %}
+                                                                    </tr>
+                                                                    <tr>
+                                                                        <td>{% trans 'Core' %} #'s:</td>
+                                                                        {% for key in request.keydata_set.all %}
+                                                                            <td>{{ key.new_core_number }}</td>
+                                                                        {% endfor %}
+                                                                    </tr>
+                                                                    <tr>
+                                                                        <td>{% trans 'Key' %} #'s:</td>
+                                                                        {% for key in request.keydata_set.all %}
+                                                                            <td>{{ key.lost_key_number }}</td>
+                                                                        {% endfor %}
+                                                                    </tr>
+                                                                    <tr>
+                                                                        {% comment %}
+                                                                        <td>{% trans 'Comments' %}:</td>
+                                                                        {% for key in request.comment_set.all %}
+                                                                            <td>{{ comment.message }}</td>
+                                                                        {% endfor %} {% endcomment %}
+                                                                    </tr>
+                                                                </tbody>
+                                                            </table>
+                                                        </td>
+                                                    </tr>
+                                                </tbody>
+                                            </table>
+                                            <br>
+                                            {% url 'edit-request' pk=request.pk as edit_url %}
```

Comment:
Uhh, I believe we did it this way so it wouldn't interfere with the blocktrans tag. It seems a little weird, but I think if you do it the other way, it won't work.